models/issues/milestone_list.go

```diff
@@ -78,8 +78,10 @@ func GetMilestones(opts GetMilestonesOption) (MilestoneList, int64, error) {
 		sess.Desc("num_issues")
 	case "id":
 		sess.Asc("id")
+	case "name":
+		sess.Desc("name")
 	default:
-		sess.Asc("deadline_unix").Asc("id")
+		sess.Asc("name").Asc("id")
```

Comment:
I am against that change.
That is not how it should behave.
What I can agree on, however, is
```suggestion
		sess.Asc("deadline_unix").Asc("name")
```

Reply:
I am unsure about sorting by name as you run into `v1 ... v19 v2 v20` trouble as it's not aware of version numbers.